### PR TITLE
Refactor Routes

### PR DESCRIPTION
**Pull request by santipalenque**
refactoring of routes logic so we use the lib as we should and it makes sense

### CODE REVIEW

File: src/routes/WithAuthRoute.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  // we are checking credentials if userProfile is being loading yet or
+  // if we are refetching the user profile to check new data ( user currently is not a authz
+  if (!userIsReady() || checkingCredentials()) {
+    return <HeroComponent title="Checking credentials..." />;
+  }
+
+  // if we have the user profile but has no tickets for the show
+  if (!userIsAuthz() && userIsReady()) {
+    setTimeout(() => { navigate(location.state?.previousUrl || "/") }, 3000);
+    return (<HeroComponent title="You are not authorized to view this session!" />);
```
Comment (smarcet) [inline src/routes/WithAuthRoute.js:64]: this is not correct
please check https://github.com/fntechgit/virtualevent-poc/blob/87c6ab97843770daba94ce29129ff6f8fd7f6d22/src/routes/PrivateRoute.js#L77
we need to redirect to page https://github.com/fntechgit/virtualevent-poc/blob/main/src/pages/authz/ticket.js to continue with the redirection flow to registration

**Review by smarcet — CHANGES_REQUESTED** [review]
please review comments and remove *.json files
regards

File: src/routes/WithAuthRoute.js (hunk 1 above)
Comment (santipalenque) [reply src/routes/WithAuthRoute.js]: got it, added redirect back in . Removed withTicket route since it is covered inside withAuth

**Comment by santipalenque** [comment]
done @smarcet please check again

**Review by smarcet — APPROVED** [review]
looks good to me

**Pull request merged**